localstack/testing/scenario/provisioning.py

```diff
@@ -14,6 +16,7 @@
 
 LOG = logging.getLogger(__name__)
 CDK_BOOTSTRAP_PARAM = "/cdk-bootstrap/hnb659fds/version"
+WAITER_CONFIG = {"Delay": 10, "MaxAttempts": 1000}
```

Comment:
I think we should have two configs here, one for AWS and one for localstack since a delay of 10s is probably unnecessary in our CI runs